components/bounty/page-actions/page-actions.d.ts

```diff
@@ -2,18 +2,13 @@ import { IssueBigNumberData } from "interfaces/issue-data";
 
 export interface PageActionsViewProps {
   handleEditIssue: () => void;
-  handlePullrequest: (arg: {
-    title: string;
-    description: string;
-  }) => Promise<void>;
+  handlePullrequest: () => void;
```

Comment:
maybe have a more friendly name, "onCreateDeliverableClick" would help in the future